src/diff.ts

```diff
@@ -174,7 +174,7 @@ export function summary(
 ${DIFF_COMMENT_IDENTIFIER}
 ${DIFF_HASH_COMMENT_TEMPLATE.replace('%s', hash)}
 
-# 🔍 Neon Schema Diff summary
+# <picture><source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/neondatabase/schema-diff-action/refs/heads/impr/modify-icon/docs/logos/logo-dark.svg"><img alt="Neon logo" src="https://raw.githubusercontent.com/neondatabase/schema-diff-action/refs/heads/impr/modify-icon/docs/logos/logo-light.svg" width="24" height="24"></picture> Neon Schema Diff summary
```

Comment:
Probably you should upload the icon first in different PR and then use the URL. I am guessing when this PR is merged the branch will be deleted and the ref `refs/heads/impr/modify-icon` might also break.

Reply:
That was supposed to be from main 🤦🏼